app/src/main/java/com/kylecorry/trail_sense/astronomy/ui/AstroChart.kt

```diff
@@ -85,19 +78,15 @@ class AstroChart(chart: Chart, private val onImageClick: () -> Unit) {
         emptyList(),
         bitmapLoader.load(R.drawable.ic_sun, imageSize.toInt()),
         16f,
-    ) {
-        onImageClick()
-        true
-    }
+        onImageClick
```

Comment:
Can you revert this change and the other onImageClick change - this method needs to return a boolean